packages/integrations/lit/server-shim.js

```diff
@@ -3,12 +3,11 @@ import { installWindowOnGlobal } from '@lit-labs/ssr/lib/dom-shim.js';
 if (typeof fetch === 'function') {
 	const _fetch = fetch;
 	installWindowOnGlobal();
-	globalThis.fetch = window.fetch = _fetch;
+	globalThis.fetch = _fetch;
 } else {
 	installWindowOnGlobal();
 }
 
```

Comment:
I think this entire block may be unnecessary now as the dom shim is built into the node builds of `lit`.

Reply:
In runtime for some reason it does seem that `globalThis.fetch !== _fetch` for some reason, so I included it. WDYT?